website/source/docs/provisioners/ansible.html.md

```diff
@@ -128,6 +128,13 @@ Optional Parameters:
 -   `user` (string) - The `ansible_user` to use. Defaults to the user running
     packer.
 
+-   `set_winrm_passwd` (boolean) - Set to `true` if you are running on AWS, Azure or 
+    Google Compute and would like to access the generated password that Packer uses to 
+    connect to the instance via WinRM. The password will be avaliable on the builder 
```

Comment:
tiny nitpick: replace  `The password will be avaliable on the builder ` with `The password will be avaliable to the anisble provisioner ` ("builder" has a very specific meaning within Packer)